lib/factory_bot/linter.rb

```diff
@@ -65,12 +65,12 @@ def lint_factory(factory)
     def lint_traits(factory)
       result = []
       factory.definition.defined_traits.map(&:name).each do |trait_name|
-        begin
+        
           FactoryBot.public_send(factory_strategy, factory.name, trait_name)
-        rescue => error
+        rescue StandardError => error
```

Comment:
I'm really not sure why it's complaining about this. I thought it was because I removed the `begin` keyword, but now that I added it back, Hound is still complaining. I think this means that I can fix the `Style/RedundantBegin` cop after all.